lib/index.js

```diff
@@ -172,17 +172,17 @@ MapboxGeocoder.prototype = {
     var selected = this._typeahead.selected;
     if (selected) {
       if (this.options.flyTo) {
-        if (!exceptions[selected.id] && selected.bbox) {
+        if (!exceptions[selected.properties.short_code] && selected.bbox) {
```

Comment:
@scottsfarley93 is it possible for a result to not have `properties` ? I want to check because if a result didn't have `properties` then this would throw an error. If there is a doubt, we could update this (and similarly on line 178) to:

```suggestion
        if (selected.properties && !exceptions[selected.properties.short_code] && selected.bbox) {
```